plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -115,14 +107,19 @@ export class PersonState {
         return this.personContainer
     }
 
-    private async createPersonIfDistinctIdIsNew(): Promise<Person | undefined> {
+    private async createPersonIfDistinctIdIsNew(): Promise<boolean> {
+        // :TRICKY: Short-circuit if person container already has loaded person and it exists
+        if (this.personContainer.loaded) {
+            return false
```

Comment:
Note that here it's possible that another request (that did `identify` for example) also updated person properties. However this isn't something we need to fix as we don't give strong guarantees on person properties attached to the event are exact if they are frequently updated.